scripts/install_foxx.sh

```diff
@@ -149,7 +149,7 @@ fi
 url2="http://${local_DATAFED_DATABASE_HOST}:${local_DATABASE_PORT}/_api/database"
 # We are now going to initialize the DataFed database in Arango, but only if sdms database does
 # not exist
-output=$(curl -s -i --dump - --user "$basic_auth" "$url2")
+output=$(LD_LIBRARY_PATH="${DATAFED_DEPENDENCIES_INSTALL_PATH}:$LD_LIBRARY_PATH" curl -s -i --dump - --user "$basic_auth" "$url2")
```

Comment:
Do we want to keep dump here?

Reply:
That is addressed in this earlier PR. https://github.com/ORNL/DataFed/pull/1188

Reply:
But we haven't merged it, why are we keeping `--dump -` if we know it causes issues?

Reply:
Let's get it merged and update this PR

Reply:
It will be merged, I just hadn't changed the target branch.